### ISSUE TITLE
Suggestion: e-sports

### ISSUE THREAD

**Issue by Th-byte** [binogure-studio/city-game-studio-i18n#319]
Esports can be created for games like league of legends and starcraft (MOBA), and each company can have its own team competing.

**Comment by xsellier** [comment]
Yes, but this is off-topic, and Im not gonna implement that feature.

**Comment by Th-byte** [comment]
Thank you for taking the time to reply to me, but what I want to express is that realizing this function will make the gameplay more diversified and make the game more interesting.
